.github/workflows/ci.yml

```diff
@@ -2,15 +2,27 @@
 # SPDX-FileCopyrightText: 2021 The Elixir Team
 # SPDX-FileCopyrightText: 2012 Plataformatec
 
-name: CI
+name: Elixir Building and Testing
 
 on:
   push:
-    paths-ignore:
-      - "lib/**/*.md"
+    paths: &paths-filter
```

Comment:
Can you please revert the precise "paths"? Those other files change rarely and one day we may add a markdown fixture that actually changes the test, and we will spend a long time without noticing it. Unless the files are common and change frequently and they do non't need to run (the case for `lib/*/pages/**/*.md`, we shouldn't worry about excluding them).